client/src/components/Allocations/AllocationsModals/AllocationsManageTeamTable/AllocationsManageTeamTable.jsx

```diff
@@ -35,14 +26,33 @@ const AllocationsManageTeamTable = ({ rawData, projectId }) => {
       {
         Header: 'Role',
         accessor: ({ role }) => {
-          switch (role) {
-            case 'PI':
-              return 'Principal Investigator';
-            case 'Delegate':
-              return 'Allocation Manager';
-            default:
-              return 'Member';
-          }
+          /*const changeUserRole = (user,role) => {
+            dispatch({
+              type: ##UPDATE_USER_ROLE_IN_TAS_PROJECT,
+              payload: user, role
+            })
+          }*/
+          const allocationRoles = {
+            Standard: 'Member',
+            Delegate: 'Allocation Manager',
+            PI: 'Principal Investigator',
+          };
+          console.log(role);
+          return (
+            <div>
+              <DropdownSelector
+                //onChange={(e) => changeUserRole(user, e.target.value)}
+                value=""
+              >
+                <option value="">{allocationRoles[role]}</option>
+                {Object.keys(allocationRoles)
+                  .filter((userRole) => userRole !== role)
+                  .map((userRole) => (
+                    <option value="">{allocationRoles[userRole]}</option>
+                  ))}
+              </DropdownSelector>
```

Comment:
This is to be implemented later right? If so, let's `disable` the selector, and just show the role for now